CONTRIBUTING.md

```diff
@@ -210,6 +168,46 @@ By default the build will use the native image server. This speeds up the build,
 not being invalidated correctly in some cases. To run a build with a new instance of the server you can use
 `./mvnw install -Dnative-image.new-server=true`.
 
+### Workflow tips
+
+Due to Quarkus being a large repository, having to rebuild the entire project every time a change is made isn't very productive. 
+The following Maven tips can vastly speed up development when working on a specific extension.
+
+#### Building all modules of an extension
+
+Let's say you want to make changes to the `Jackson` extension. This extension contains the `deployment`, `runtime` and `spi` modules
+which can all be built by executing following command:
+
+```
+./mvnw install -f extensions/jackson/
+```
+
+This command uses the path of the extension on the filesystem to identify it. Moreover, Maven will automatically build all modules in that path recursively.
+
+#### Building a single module of an extension
+
+Let's say you want to make changes to the `deployment` module of the Jackson extension. There are two ways to accomplish this task as shown by the following commands:
+
+```
+./mvnw install -f extensions/jackson/deployment
+```
+
+or 
+
+```
+./mvnw install --projects 'io.quarkus:quarkus-jackson-deployment'
```

Comment:
Seen too late, but you can also use: `-pl extensions/jackson/deployment` which is especially nice to type when you want to select multiple projects (shell path completion to the rescue).
(For single projects `-f` is superior since it doesn't parse all `pom.xml` files.)

PS: `-pl :quarkus-jackson-deployment` works as well.

Reply:
We have a similar tip, no?

Reply:
Do we? Where? Not sure I am following...

Reply:
https://github.com/quarkusio/quarkus/blob/6ddcf18d0c9812a055c3ade2365fe9acc063d75f/CONTRIBUTING.md#L192

Reply:
Yeah ok, that's `-f`. My main point is that you can use much short expressions for/with `-pl` which is useful when you want to build _multiple_ modules together.